config/config.txt

```diff
@@ -26,7 +26,8 @@ $include lua.txt
 # DISCORD_LINK https://discord.gg/hGpZ4Z3
 
 ## Server SQL name: This is the name used to identify the server to the SQL DB, distinct from SERVERNAME as it must be at most 32 characters.
-# SERVERSQLNAME tgstation
+## Useful if you intend to run multiple servers that speak to the same database to share staff ranks, bans, notes etc.
+SERVERSQLNAME
```

Comment:
Don't change the values in the config from upstream, that's the best way to get some merge conflicts to slip under the radar later on.
```suggestion
# SERVERSQLNAME tgstation
```